arq/worker.py

```diff
@@ -186,6 +186,7 @@ def __init__(
         on_job_start: Optional['StartupShutdown'] = None,
         on_job_end: Optional['StartupShutdown'] = None,
         handle_signals: bool = True,
+        use_k8s_signal_handler: bool = False,
```

Comment:
This could be renamed to `cancel_job_on_sigint` or something like that? And then we could generalize the rest.

Reply:
yes agreed.

Reply:
Thanks! I'll finalize when I'm back from my vacation.